### PR TITLE
Change changelog for rss feed

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: layouts/docs/changelog.html [hunk 1]
```diff
@@ -3,32 +3,70 @@
...
-                  <button class="filter-button" onclick="filterSelection('changelog-improved')" id="improved-button">Improvements</button>
-                  <button class="filter-button" onclick="filterSelection('changelog-added')" id="added-button">Additions</button>
-                  <button class="filter-button" onclick="filterSelection('changelog-changed')" id="changed-button">Changes</button>
-                  <button class="filter-button" onclick="filterSelection('changelog-removed')" id="removed-button">Removals</button>
-                  <button class="filter-button active" onclick="filterSelection('all')" id="all-button">Show All</button>
+                  <button
+                    class="filter-button"
+                    onclick="filterSelection('changelog-improved')"
+                    id="improved-button"
+                  >
```
Comment (JessamyT) [inline layouts/docs/changelog.html:39]: Why are these all stretched out? Is this how it should be formatted?

File: layouts/docs/changelog.html [hunk 2]
```diff
@@ -3,32 +3,70 @@
...
+                    Additions
+                  </button>
+                  <button
+                    class="filter-button"
+                    onclick="filterSelection('changelog-changed')"
+                    id="changed-button"
+                  >
+                    Changes
+                  </button>
```
Comment (JessamyT) [inline layouts/docs/changelog.html:55]:
```suggestion
                  >Changes</button>
```
[opt/just wondering] Wouldn't at least this much condensation actually make it more readable and consistent with other tags?

File: layouts/docs/tutorials.html
Comment (JessamyT) [inline layouts/docs/tutorials.html:1]: Is this related to changelog?

File: layouts/shortcodes/changelog.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<item>
+  <title>{{ .Get "title" }}</title>
+  <link>https://docs.viam.com/appendix/#{{ .Get "title" | anchorize }}</link>
+  <pubDate>{{ .Get "date" | time.Format "Mon, 02 Jan 2006 15:04:05 -0700" }}</pubDate>
```
Comment (JessamyT) [inline layouts/shortcodes/changelog.xml:4]: Not sure how the date formatting works but, this is different from the format passed into the shortcodes in changelog.md...how does this work? And do we really need to have a date instead of just a month, when the date is not actually the first of the month?

File: docs/appendix/changelog.md [hunk 4]
```diff
@@ -21,52 +22,48 @@ Check out [this tutorial](/tutorials/projects/filtered-camera/) for more informa
 
 {{% /changelog %}}
 
-{{% changelog color="added" title="Raspberry Pi 5 Support" %}}
+{{% changelog date="2023-12-01" color="added" title="Raspberry Pi 5 Support" %}}
```
Comment (JessamyT) [inline docs/appendix/changelog.md:25]: If possible I think we shouldn't put the first of the month since that's not accurate

File: layouts/docs/changelog.html (hunk 1 above)
Comment (npentrel) [reply layouts/docs/changelog.html:39]: I think my editor possibly automatically formatted this

File: layouts/shortcodes/changelog.xml (hunk 3 above)
Comment (npentrel) [reply layouts/shortcodes/changelog.xml:4]: This is the format RSS expects. The commands change the formatting to match

Comment (npentrel) [reply layouts/shortcodes/changelog.xml:4]: I think we do need to add a full date but I'll test tomorrow

Comment (JessamyT) [reply layouts/shortcodes/changelog.xml:4]: Okay. I don't love the idea that someone could get the RSS feed and think that they should be looking for a viam-server version that came out the first of the month

Comment (JessamyT) [reply layouts/shortcodes/changelog.xml:4]: If it's not possible to eliminate the full date day, do you think it'd be worth it to (in a separate ticket) fix these with more accurate dates? We used to have most of them in here anyway I think, so could just re-apply them

File: docs/appendix/changelog.md (hunk 4 above)
Comment (npentrel) [reply docs/appendix/changelog.md:25]: changed to the end of the month. I don't think any dates except the vision service are accurate. But we did used to announce everything at the end of the month so may as well stick with that

File: layouts/docs/changelog.html (hunk 1 above)
Comment (npentrel) [reply layouts/docs/changelog.html]: changed back

File: docs/appendix/changelog.md [hunk 5]
```diff
@@ -692,24 +679,22 @@ You will need to first register the machine learning model file with the [ML mod
 
 {{% /changelog %}}
 
-## March 2023
-
-{{% changelog color="added" title="Machine learning for image classification models" %}}
+{{% changelog date="2023-03-31" color="added" title="Machine learning for image classification models" %}}
 
 You can now [train](/ml/train-model/) and [deploy](/ml/deploy/#create-an-ml-model-service) image classification models with the [data management service](/data/) and use your machine's image data directly within Viaml
```
Comment (JessamyT) [inline docs/appendix/changelog.md:684]:
```suggestion
You can now [train](/ml/train-model/) and [deploy](/ml/deploy/#create-an-ml-model-service) image classification models with the [data management service](/data/) and use your machine's image data directly within Viam.
```

**Review by JessamyT — APPROVED** [review]
LGTM! Noticed an out-of-diff nit if you care to fix
